xbmc/platform/win32/network/WSDiscoveryWin32.cpp

```diff
@@ -138,7 +139,7 @@ HRESULT STDMETHODCALLTYPE CClientNotificationSink::SearchFailed(HRESULT hr, LPCW
   CSingleLock lock(m_criticalSection);
 
   // This must not happen. At least localhost (127.0.0.1) has to be found
-  CLog::Log(LOGWARNING,
+  CLog::Log(LOGWARNING, LOGWSDISCOVERY,
```

Comment:
I would like to leave this log visible as it was before since it is something that normally never has to happen, so it is a warning that something is wrong.

Reply:
no worries, will change it back